packages/jaeger-ui/src/components/TracePage/CriticalPath/utils/sanitizeOverFlowingChildren.tsx

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2023 The Jaeger Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Span } from '../../../../types/trace';
+
+/**
+ * This function takes an array of spans and sanitizes them by handling overflow
```

Comment:
> sanitizes them by handling overflow scenarios

see, this doesn't actually explain what it's doing. What is "sanitizing" and "handing"?

Reply:
And is this global transformation actually necessary? Why can't we handle the overflow use case directly in the LFC function?

Reply:
> > sanitizes them by handling overflow scenarios
> 
> see, this doesn't actually explain what it's doing. What is "sanitizing" and "handing"?

https://github.com/jaegertracing/jaeger-ui/blob/36ce16b582d0a54d300014e9e228dc7cff21e772/packages/jaeger-ui/src/components/TracePage/CriticalPath/utils/sanitizeOverFlowingChildren.tsx#L17-L26
Is this function definition fine for sanitizeOverFlowingChildren function??